### ISSUE TITLE
Compressed ASR creates invalid message

### ISSUE THREAD

**Issue by no-man-is-an-island** [NHSDigital/integration-adaptor-gp2gp#400]
Affects version: 

gp2gp: 1.3.1
MHS outbound: 1.1.1

Description:

When sending a large ASR for outbound the following validation error is triggered:

```
[2021-10-19T19:58:42.538514Z] | ERROR | 7 | RCMR_IN030000UK06 | 472535ED-99C2-4483-9D84-05110C89D327 | 154a5341-ce62-4ef9-889a-999786baca44 | | outbound.outbound.request.synchronous.handler | Invalid request. ValidationErrors="{'attachments': {0: {'description': ['Length must be between 1 and 100.'], 'is_base64': ['Not a valid boolean.']}}}"
```

In the message that has been sent, the following fields are suspect:

```
"is_base64":"false" (string not bool)
```

```
"description":"\n                Filename=154a5341-ce62-4ef9-889a-999786baca44/8FA3D5CA-5A48-4DF4-8FF5-BDB0CD0A8DA9.json\n                ContentType=application/xml\n                Compressed=Yes\n                LargeAttachment=Yes\n                OriginalBase64=No\n                Length=1107529\n                DomainData=X-GP2GP-Skeleton: Yes\n            \"
```

**Comment by no-man-is-an-island** [comment]
It looks like while this ASR is large, it doesn't need to split into multiple messages as it compresses into less than 5MB

**Comment by adrianclay** [comment]
This was fixed in #440 and #461
